app/src/main/java/com/hammer/talkbbokki/domain/usecase/TopicUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package com.hammer.talkbbokki.domain.usecase
+
+import com.hammer.talkbbokki.domain.model.TopicItem
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between